docs/content/posts/open-source-introduction.md

```diff
@@ -0,0 +1,204 @@
+---
+date: 2019-12-26
```

Comment:
The date of the [initial commit](https://github.com/avito-tech/avito-android/commit/bc52d94ffc4121e4a901112fb77ecf3660d7f221)